scrunch/datasets.py

```diff
@@ -1882,3 +1989,56 @@ def move(self, path, position=-1):
             )
         target_group = self.dataset.order[str(path)]
         target_group.insert(self.alias, position=position)
+
+    def summary(self, names=False, tablefmt='psql'):
```

Comment:
I'm not sure about allowing different formats here, maybe lets start with one default format (whatever you think is best) and then see if there is demand for others?